docs/assets/css/custom-full.css

```diff
@@ -1,6 +1,6 @@
 ﻿/*!
- * Bootstrap  v5.2.2 (https://getbootstrap.com/)
+ * Bootstrap  v5.2.1 (https://getbootstrap.com/)
```

Comment:
We're downgrading bootstrap?

Reply:
```bash
npm --package-lock-only ls
```

will tell you what is currently installed from the package lock.

I don't really understand the benefit of pinning - if we use `ci` it uses the package lock anyway, which is always pinned.

Reply:
```
@talis/bootstrap-theme@3.0.7 /Users/david/talis/bootstrap-theme
├── @11ty/eleventy@2.0.1
├── @commitlint/cli@17.6.1
├── @commitlint/config-conventional@17.6.1
├── autoprefixer@10.4.14
├── husky@8.0.3
├── lint-staged@13.2.1
├── nodemon@2.0.22
├── npm-run-all@4.1.5
├── postcss-cli@10.1.0
├── postcss@8.4.23
├── prettier@2.8.8
├── purgecss@5.0.0
├── sass@1.62.0
├── serve@14.2.0
├── standard-version@9.5.0
├── stylelint-config-twbs-bootstrap@7.0.0
└── stylelint@15.6.0
```

Reply:
I agree with you on that we should use `ci`, but developers may opt for `i`. Pinning them would ensure that we avoid encountering a situation similar to our current one, where the lock file contains higher versions than those specified in the json.

Reply:
I've approved the changes.

Might be worth following up with enforcing [save-exact](https://docs.npmjs.com/cli/v10/using-npm/config#save-exact) to cover future packages.